dynamic-config/config-apollo/pom.xml

```diff
@@ -43,6 +43,7 @@
     <dependency>
       <groupId>javax.validation</groupId>
       <artifactId>validation-api</artifactId>
+      <version>2.0.1.Final</version>
```

Comment:
should declare in DM, and already declared in DM
what caused declare again in this place to resolve problem?